modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/config/ConfigFactory.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.ignite.cli.config;
+
+import io.micronaut.context.annotation.Factory;
+import jakarta.inject.Singleton;
+
+@Factory
+public class ConfigFactory {
+    @Singleton
+    public Config createConfigFromFile() {
```

Comment:
factory method should be named without verb, it is a bean name. "fileConfig" would be beter